### DISCUSSION TITLE
AVD 4.7.1 Banner

### DISCUSSION THREAD

**Discussion by pradeeprgodhala** [aristanetworks/avd#3951] [Data Model Schemas] [answered]
Hi,

I am running 4.7.1 AVD and i am trying to use Banner Motd, but its not generating multi-line Banner even though the yaml input is multi-line input.
#YAML Input#

```yaml
banners:
  motd: '
  
   
  sahkjfds
  sfbsdf
  sdfbk
  lndfsd
  
  
  
  EOF'
```
#Config Generated#

```eos
banner motd
|

sahkjfds sfbsdf sdfbk lndfsd


EOF
!
```

What do i need to yaml input to get this as multi-line input.

**Comment by ClausHolbechArista** [comment, marked as answer]
I updated your issue with code blocks to make it easier to follow.

The issue here is YAML syntax. When you use the single quotes, you have to add double new-lines to produce a single new-line.

I would suggest the following:

```yaml
banners:
  motd: |2-
    Multiline
    test with spaces on
       one line
    EOF
```
- `|` means multiline input, where each new line is also a new line in the output.
- `2` means the input is interpreted with 2 spaces as indentation, so any extra spaces are included in the output.
- `-` means the final newline will be stripped (prevents an extra blank line after the EOF)

Please let me know if this works for you.

**Reply by pradeeprgodhala** [reply]
Thank you... This suggestion worked.
